.travis.yml

```diff
@@ -32,6 +32,9 @@ before_install:
 - if [ "$TRAVIS_OS_NAME" = "linux" ]; then USE_NINJA="-GNinja"; VALGRIND="valgrind $VALGRIND_OPTIONS"; . /opt/qt57/bin/qt57-env.sh; fi
 
 install:
+- pushd 3rdparty/libQtOlm
+- git clone https://gitlab.matrix.org/matrix-org/olm.git
+- popd
```

Comment:
Please add the same to `.appveyor.yml` under `before_build` (except that pushd and popd don't work there, so replace them with ordinary cd).